### PR TITLE
Use QR code instead of sms

### PR DESCRIPTION
**Pull request by jkmassel**
Removes SMS functionality in favour of a QR code.

Visit [/me/get-apps](https://wordpress.com/me/get-apps) and test the functionality. The QR code should send you to https://apps.wordpress.com/get/?campaign=calypso-qrcode-apps

### CODE REVIEW

File: client/blocks/get-apps/mobile-download-card.jsx [hunk 1]
```diff
@@ -188,45 +184,20 @@ class MobileDownloadCard extends Component {
...
-							) : (
-								<>
-									<FormPhoneInput countriesList={ this.props.countriesList } isDisabled={ true } />
-								</>
-							) }
-						</div>
-						<div className="get-apps__sms-button-wrapper">
-							<Button
-								className="get-apps__sms-button"
-								onClick={ this.onSubmit }
```
Comment (enejb) [inline client/blocks/get-apps/mobile-download-card.jsx:222]: We can probably also delete some of these `onSubmit, onKeyUp, onChange` actions that are found in this deleted code.

Comment (jkmassel) [reply client/blocks/get-apps/mobile-download-card.jsx:222]: Yes, agreed – I took a shot at it before and ended up breaking things. If I super-promise to come back to this, could we land this now and land a separate cleanup PR next?

Comment (enejb) [reply client/blocks/get-apps/mobile-download-card.jsx:222]: Sure.

**Review by enejb — APPROVED** [review]
The changes work well and things look as expected.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7227233

Hi @jkmassel, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: `Scan the code using your mobile phone to download the app.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7227233).

**Pull request merged**